### ISSUE TITLE
cpus does not work for moda_style_1

### ISSUE THREAD

**Issue by cguetot** [ursgal/ursgal#130]
Hi guys,

The parameter `cpus` is not working when I run MoDa.   I know the wrapper should activate `-@` on MoDA. I already tried from a fresh install on Linux but the behavior remains the same. 

I hope it has an easy fix.

thanks,
C

**Comment by StSchulze** [comment]
Hi Carlos,

I'm sorry, I guess I had missed that when adding the MoDA v.1.6.1 because it wasn't an option before that. However, it also seems like there were some problems with it, because the new version (1.6.2) has "Corrected thread issues".
So, I will implement version 1.6.2 as soon as I get the download link, so that I can also test it, and add the cpu option.

Best,
Stefan

**Comment by StSchulze** [comment]
I have added version 1.62 in the branch 'hotfix/moda_v1_62', however, it doesn't work with my test files, throwing the following error:

> 
> [ prefligh ] Execution time 0.000 seconds
> [ eXecutio ] Executing command list ...
> *********************************************************
> MODa v1.62: Multi-Blind Modification Search
> Release Date: Jun 12, 2019
> Hanyang University, Seoul, Korea
> *********************************************************
> 
> Reading parameter.....
> Input datasest : ..\example_data\test\Haloferax_volcanii_FPF_ArcPP_20190606_unipro_cRAP_target_decoy_trypsin.fasta
> Input datasest : C:\Users\Admin\Desktop\ursgal\example_data\test\StS_Hvo_0405_cytosol_B_FASP_v5_26102016.mgf
> Fixed Mod : 57.021464 is added to amino acid 'C'
> 
> Starting MOD-Alignment for multi-blind modification search!
> Reading MS/MS spectra.....  11729 scans
> Reading protein database.....  8538 proteins / 2391442 residues (1)
> 
> Abnormal Termination

This doesn't seem to be the result of some coding mistake within Ursgal, as using the same wrapper for v1.61 works perfectly fine.

So, @cguetot could you checkout the hotfix and try both versions with your files?

**Comment by MKoesters** [comment]
I get the same error

```[ prefligh ] Execution time 0.000 seconds
[ eXecutio ] Executing command list ...
*********************************************************
MODa v1.62: Multi-Blind Modification Search
Release Date: Jun 12, 2019
Hanyang University, Seoul, Korea
*********************************************************

Reading parameter.....
Input datasest : /home/manuel/Data/Proteomics/databases/uniprot_sprot_human_CRAP_target_decoy.fasta
Input datasest : /home/manuel/Gits/Ursgal/ursgal_github2/tests/data/test_Creinhardtii_QE_pH8.mgf

Starting MOD-Alignment for multi-blind modification search!
Reading MS/MS spectra.....  244 scans
Reading protein database.....  41952 proteins / 22971544 residues (1)

Abnormal Termination
```

**Comment by cguetot** [comment]
I also got the `Abnormal Termination` message. 

Should I close the issue ?

**Comment by StSchulze** [comment]
Did version 1.61 work for you?
If yes, the issue can be closed and the fix will be merged.
In any case, I'll contact the MoDa developers for the Abnormal Termination issue.

**Comment by StSchulze** [comment]
After contacting the developers, I have now fixed the issue with #131 
Please check it out - for now I will close this issue, but feel free to open it again if there are still problems!

Quick summary of what caused the problem:
with v1.62 the output file behavior was changed, so that the -o flag in MODa cannot specify a filename anymore, but only a folder, the filename is then generated automatically.
